build/ci_utils/src/cache/goodie/graalvm.rs

```diff
@@ -166,14 +158,49 @@ mod tests {
     /// Check that we correctly recognize both the GraalVM version and the Java version.
     #[test]
     fn version_recognize() {
-        let version_string = r"openjdk version 17.0.6 2023-01-17
-OpenJDK Runtime Environment GraalVM CE 22.3.1 (build 17.0.6+10-jvmci-22.3-b13)
-OpenJDK 64-Bit Server VM GraalVM CE 22.3.1 (build 17.0.6+10-jvmci-22.3-b13, mixed mode, sharing)";
+        let version_string = "openjdk version \"17.0.7\" 2023-04-18
```

Comment:
How could we remove these hardcoded versions from here and reuse those already defined in `build.sbt`? Could we create a `sbt verify-proper-graalvm` task and invoke it on the downloaded JVM? Such an `sbt` task could easily re-use the version information from `build.sbt`.

Reply:
I don't understand. This is just a Rust unit test that does not have anything to do with the actual version of GraalVM used, i.e, with the version specified in `build.sbt`.